sky/sdk/lib/widgets/animated_container.dart

```diff
@@ -53,6 +53,22 @@ class AnimatedEdgeDimsValue extends AnimatedValue<EdgeDims> {
   }
 }
 
+class AnimatedMatrix4Value extends AnimatedValue<Matrix4> {
+  AnimatedMatrix4Value(Matrix4 begin, { Matrix4 end, Curve curve: linear })
+    : super(begin, end: end, curve: curve);
+
+  void setProgress(double t) {
+    if (t == 1.0) {
+      value = end;
+      return;
+    }
+    Vector3 beginT = begin.getTranslation();
+    Vector3 endT = end.getTranslation();
+    Vector3 lerpT = beginT*(1.0-t) + endT*t;
+    value = new Matrix4.identity()..translate(lerpT);
```

Comment:
Oh, yeah, it's definitely a temporary solution. I just wanted something that works for now. I'll add a TODO.